### PR TITLE
Ensure param block pointer not clobbered

### PR DESCRIPTION
**Pull request by alilee**
Got one!

### CODE REVIEW

File: src/aarch64.rs [hunk 1]
```diff
@@ -28,13 +28,13 @@ struct qemu_parameter_block {
...
     unsafe {
-        asm!(
+        llvm_asm!(
             "mov w0, 0x18
              mov x1, $0
              hlt #0xF000"
              : // No Outputs
              : "r"(block as *const _ as u64)
-             : "memory"
+             : "memory", "w0"
```
Comment (andre-richter) [inline src/aarch64.rs:37]: Good catch. 
While we are at it, `x1` is also altered by the assembly.

So it should be the following?

```suggestion
             : "w0", "x1", "memory"
```

Comment (alilee) [reply src/aarch64.rs]: I concur.

File: src/lib.rs [hunk 2]
```diff
@@ -51,6 +51,7 @@
 #![feature(asm)]
```
Comment (andre-richter) [inline src/lib.rs:51]:
```suggestion

```

**Comment by andre-richter** [comment]
Thanks!

**Comment by andre-richter** [comment]
Released as `0.1.1`

**Pull request merged**